packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -98,11 +98,20 @@ export default {
     RecipientContainer,
     ExpirationDatepicker
   },
+  props: {
+    savingButtonTitle: {
```

Comment:
prop name is a bit weird. `saveButtonLabel` would be more appropriate (`save`  instead of `saving` because it reads nicer and `Label` instead of `Title` because it actually is the label of the button).